docs/specification/2025-03-26/basic/authorization.md

```diff
@@ -4,242 +4,394 @@ type: docs
+   unusual cases for authentication and authorization may emerge (e.g., mTLS, Kerberos), which will
+   need clients to be extensible.
+
+### 1.3 Protocol Requirements
 
 Authorization is **OPTIONAL** for MCP implementations. When supported:
 
-- Implementations using an HTTP-based transport **SHOULD** conform to this specification.
-- Implementations using an STDIO transport **SHOULD NOT** follow this specification, and
-  instead retrieve credentials from the environment.
-- Implementations using alternative transports **MUST** follow established security best
-  practices for their protocol.
+- Streamable HTTP or SSE transports **SHOULD** conform to this specification.
+- Implementations using an STDIO transport **SHOULD NOT** follow this specification and instead
+  retrieve credentials from the environment.
+- Implementations using alternative transports **MUST** follow established security best practices for
+  their protocol.
 
-### 1.3 Standards Compliance
+### 1.4 Standards Compliance
 
-This authorization mechanism is based on established specifications listed below, but
-implements a selected subset of their features to ensure security and interoperability
-while maintaining simplicity:
+This authorization mechanism is based on established specifications listed below, but implements a
+selected subset of their features to ensure security and interoperability while maintaining simplicity:
 
-- [OAuth 2.1 IETF DRAFT](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-v2-1-12)
+- [OAuth 2.1 IETF DRAFT](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-v2-1-12)  
 - OAuth 2.0 Authorization Server Metadata
-  ([RFC8414](https://datatracker.ietf.org/doc/html/rfc8414))
+  ([RFC8414](https://datatracker.ietf.org/doc/html/rfc8414))  
 - OAuth 2.0 Dynamic Client Registration Protocol
-  ([RFC7591](https://datatracker.ietf.org/doc/html/rfc7591))
+  ([RFC7591](https://datatracker.ietf.org/doc/html/rfc7591))  
+- OAuth 2.0 Demonstrating Proof of Possession (DPoP)
+  ([RFC9449](https://datatracker.ietf.org/doc/html/rfc9449))  
+- Resource Indicators for OAuth 2.0
+  ([RFC8707](https://www.rfc-editor.org/rfc/rfc8707.html))
+
+This specification **DOES NOT** require MCP server developers to implement their own OAuth Provider
+(OP) and instead relies on developers adopting dedicated and well-tested third-party Identity
+Providers (IdPs).
+
+This specification also enables existing deployments to leverage their current infrastructure.
+As a side-effect, this also simplifies consent flows, and services that have multiple MCP Servers
+won't require individual registration per client:server pair.
 
 ## 2. Authorization Flow
 
 ### 2.1 Overview
 
-1. MCP auth implementations **MUST** implement OAuth 2.1 with appropriate security
-   measures for both confidential and public clients.
+1. Identity providers **MUST** implement OAuth 2.1 with appropriate security measures for both
+   confidential and public clients.
```

Comment:
perhaps add a reference to the OAuth Security Best Practices specification RFC 9700 (https://datatracker.ietf.org/doc/html/rfc9700) as guidance for "appropriate security measures"